services/frontend-v3/src/components/searchBar/SearchBarView.jsx

```diff
@@ -1,15 +1,24 @@
 import React, { useState } from "react";
 import { injectIntl } from "react-intl";
-import { Form } from "antd";
-import { Icon as LegacyIcon } from "@ant-design/compatible";
+import { Form, Alert } from "antd";
+import { DownOutlined, UpOutlined } from "@ant-design/icons";
+//import { Icon as LegacyIcon } from "@ant-design/compatible";
 import "@ant-design/compatible/assets/index.css";
 import { Row, Col, Button, Card } from "antd";
 import logo from "../sideNav/logo_v2.svg";
 
 function SearchBarView(props) {
-  const [expand, setExpand] = useState(false);
+  const [setDoNothing] = useState(true);
```

Comment:
This should not work...
The useState returns an array that looks like [state, setState]. You are setting setDoNothing as the state and not the setState. 
This would work with objects but not arrays.